src/commands/createNewProject/IProjectWizardContext.ts

```diff
@@ -31,6 +31,8 @@ export interface IProjectWizardContext extends IActionContext {
     targetFramework?: string | string[];
 
     containerizedProject?: boolean;
+    // Didn't want `mcpProjectType` on `IProjectWizardContext` but all projects can have mcp extension triggers
+    mcpProjectType?: McpProjectType;
 }
 
 export interface MCPProjectWizardContext extends IProjectWizardContext {
```

Comment:
lol wtf is this...

<img width="422" height="35" alt="image" src="https://github.com/user-attachments/assets/35e22f34-5836-4152-9dfb-59c0f1a4aaac" />